superset-frontend/src/explore/components/DataTablesPane.tsx

```diff
@@ -271,7 +271,7 @@ export const DataTablesPane = ({
               height: 100%;
 
               .ant-collapse-content {
-                height: 100%;
+                height: calc(100% - 33px);
```

Comment:
Can we use `theme.gridUnit * 8` instead of `33px`?

Reply:
ye！It's better than hardcode

Reply:
done, please review, thanks!

Reply:
LGTM